src/content/en/2019/fonts.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Web Fonts
```

Comment:
Thanks for working on this @zachleat! My overall feedback would be shape this chapter into more of a narrative arc. Start with an intro providing exposition describing what web fonts are and what the objective/scope of the chapter is. And a conclusion at the end is a nice way to tie it all together and give your overall thoughts on the state of web fonts. Hope that makes sense.